libsolidity/interface/Natspec.cpp

```diff
@@ -154,7 +154,7 @@ Json::Value Natspec::devDocumentation(ContractDefinition const& _contractDef)
 	}
 
 	Json::Value events(Json::objectValue);
-	for (auto const& event: _contractDef.events())
+	for (auto const& event: _contractDef.interfaceEvents())
```

Comment:
Line `86` also needs to be updated.

Reply:
It already uses `interfaceEvents` - how do you think it should be updated?

Reply:
Oops. Sorry, that's correct.